src/views/CompanySettings.js

```diff
@@ -210,3 +200,91 @@ export default connect(
     dispatchCreateRoomForHotel: createRoomForHotel,
   },
 )(CompanySettings);
+
+const CompanySettingsOuterWrapper = styled.div`
+  margin: 0 3rem;
+  @media(max-width: 800px) {
+    margin: 0 auto;
+  }
+  h2 {
+    font-size: ${theme.fontSize.l};
+    padding: 1.5rem 0;
+  }
+`;
+
+const CompanySettingsWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+  padding: 5rem 0;
+  @media (max-width: 800px) {
+    flex-direction: column;
+    max-width: 100%;
+  }
+  
+  .company-details {
+    width: 30%;
+    @media (max-width: 800px) {
+    width:100%;
+    }
+    
+    h3 {
+      font-size: ${theme.fontSize.xs};
+      color: ${theme.color.accentPurple};
+      padding: 1.5rem 0;
+    }
+    
+    form {
+      display: flex;
+      flex-direction: column;
+      
+      input {
+        border: none;
+        border-bottom: 1px solid ${theme.color.footerText};
+        margin-bottom: 20px;
+        height: ${theme.input.height};
+        font-size: ${theme.fontSize.xs};
+        padding: 20px 0;
+        border-radius: 0;
+        &:focus {
+          outline: none;
+        }
+      }
```

Comment:
There are spaces a little higher up and suddenly formatting changes to be without spaces, would be best to keep consistent 👍